Cataclysm/Druid.lua

```diff
@@ -1598,6 +1628,13 @@ spec:RegisterAuras( {
         duration = 8,
         max_stack = 1,
     },
+    -- Total Energy increased by 20.
+    primal_madness = {
+        id = 80886,
+        remains = function() return max(buff.berserk.remains, buff.tigers_fury.remains) end,
```

Comment:
Don't try to override remains/expires here.  Manipulate the expiration time in `reset_precast` and the `handler`s for Berserk and Tiger's Fury.  Provide a default `duration` even if it's inconsistent.